File: packages/java/src/main/java/com/peruspatial/territorial/RiskLevelToScore.java
package com.peruspatial.territorial;

public final class RiskLevelToScore {
  private RiskLevelToScore() {}

  public static int riskLevelToScore(String level) {
    if (level == null) return 0;
    return switch (level) {
      case "BAJO" -> 1;
      case "MEDIO" -> 2;
      case "ALTO" -> 3;
      default -> 0;
    };
  }
}

File: packages/typescript/src/territorial/riskLevelToScore.ts
import type { RiskLevel } from "../types";

export function riskLevelToScore(level: string): number {
  switch (level as RiskLevel) {
    case "BAJO":
      return 1;
    case "MEDIO":
      return 2;
    case "ALTO":
      return 3;
    default:
      return 0;
  }
}
